lang/en.json

```
@@ -47,6 +47,9 @@
   "Title from the viewMode GSoC branch": "Title from the viewMode GSoC branch",
   "Please verify URL first": "Please verify URL first",
   "Enter a valid URL": "Enter a valid URL",
+  "No URL configured. Please set URL first.": "No URL configured. Please set URL first.",
+  "Join Organisation with QR": "Join Organisation with QR",
+  "Scan QR": "Scan QR",
```

Biome (2.1.2)

[error] The key Scan QR was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)